src/screens/journey.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { useEffect } from 'react';
+import { StyleSheet, View, } from 'react-native';
+import { Text, Button } from 'react-native-paper';
+import { config } from '../config';
+import { ScreenNavigationProps } from '../routes';
+import TicketFlatList from '../components/ticketsFlatList';
+import { JourneyOutput } from '../types/JourneyOutput';
+
+const baseUrl = 'https://mobile-api-softwire2.lner.co.uk';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#eee',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  text: {
+    paddingBottom: 24,
+  },
+});
+
+type JourneyScreenProps = ScreenNavigationProps<'Journey'>;
+
+type JourneyResponse = {
```

Comment:
It would be better to pull all of these types into a response model file, maybe at `src/model/journeyResponse.ts`